javascript/reactjs-todo/.env.example

```diff
@@ -10,4 +11,6 @@ REALM_PATH=$REALM_PATH
 REST_OAUTH_CLIENT=$REST_OAUTH_CLIENT
 REST_OAUTH_SECRET=$REST_OAUTH_SECRET
 WEB_OAUTH_CLIENT=$WEB_OAUTH_CLIENT
-CENTRALIZED_LOGIN=$CENTRALIZED_LOGIN
\ No newline at end of file
+CENTRALIZED_LOGIN=$CENTRALIZED_LOGIN
+SERVER_TYPE=$SERVER_TYPE
+WELL_KNOWN=$WELL_KNOWN
```

Comment:
When these expect a full URL, I like the variable name to explicitly state that that. So, for this, I'd suggest `WELLKNOWN_URL`.